drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -5064,6 +5064,10 @@ void RasterizerSceneGLES3::initialize() {
 
 	render_list.max_elements = GLOBAL_DEF_RST("rendering/limits/rendering/max_renderable_elements", (int)RenderList::DEFAULT_MAX_ELEMENTS);
 	ProjectSettings::get_singleton()->set_custom_property_info("rendering/limits/rendering/max_renderable_elements", PropertyInfo(Variant::INT, "rendering/limits/rendering/max_renderable_elements", PROPERTY_HINT_RANGE, "1024,1000000,1"));
+	render_list.max_lights = GLOBAL_DEF("rendering/limits/rendering/max_lights", (int)RenderList::DEFAULT_MAX_LIGHTS);
```

Comment:
The setting name doesn't match the ones below (`"rendering/limits/rendering/max_renderable_lights"`), is this intended?

Reply:
Nope. :) copy-paste error.